### ISSUE TITLE
GUI : FED : Application Header - Accessibility violations

### ISSUE THREAD

**Issue by derick-montague** [ibm-openbmc/dev#2041]
## Overview
1. b-nav-bar wraps items in a `<nav>` element. Not all elements, logout and refresh are buttons
1. b-nav-item does not allow buttons other than dropdowns for navigation menus
1. Not able to set the `<a>` element role attribute for b-nav-item to fix accessibility issue
1. aria-expanded is correctly set, but there isn’t an associates aria-controls linking the menu and the button nav-trigger button

## Tasks

- [x] Resolve invalid markup in header created by the nesting a`b-nav` within  `b-nav-bar-nav' component. 
- [x] Remove aria-expanded since the button is hidden from screen readers as menu is never visibly hidden

### Screenshots
![image](https://user-images.githubusercontent.com/1183515/76362672-6df99380-62ef-11ea-9287-9713b54fe5d2.png)

**Comment by derick-montague** [comment]
Resolved by: https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/31022
